### PR TITLE
Fixes #38 - Hide reports that are labelled as invalid only with high confidence

### LINKED ISSUE #38: Hide reports that are labelled as invalid only with high confidence
Let's set 95% as a confidence threshold

### PR DESCRIPTION
**Pull request by ksy36**

### CODE REVIEW

File: src/server/helpers/user_reports_transform.ts [hunk 1]
```diff
@@ -57,8 +57,12 @@ export async function fetchUserReports(projectId: string, paramFrom: string, par
...
         WHERE
           reports.submission_timestamp BETWEEN TIMESTAMP(?) and TIMESTAMP(DATE_ADD(?, interval 1 day))
-        ORDER BY CHAR_LENGTH(comments) DESC
-        ;
+        ORDER BY
+        CASE
+            WHEN label = 'invalid' AND probability >= 0.92 THEN 1
+            ELSE 0
+        END,
+        CHAR_LENGTH(comments) DESC;
```
Comment (ksy36) [inline src/server/helpers/user_reports_transform.ts:65]: Excluded reports that are labelled as invalid with confidence > 95% below. 

Thought maybe to place any invalid reports with confidence higher than 92% at the bottom of the list, so they still will be displayed, but after everything else and with lower opacity 

92% here is rather arbitrary, based on looking at reports prediction for the past week. Maybe if it's something we can experiment with :) 

wdyt @denschub ?

Comment (ksy36) [reply src/server/helpers/user_reports_transform.ts:65]: Relevant code in the component UI: 
https://github.com/mozilla/webcompat-dashboard/blob/e55206e456ac7b9b8ea0be9e5b797d4ae16cdfbe/src/client/components/user_report_contents.tsx#L21

Comment (denschub) [reply src/server/helpers/user_reports_transform.ts:65]: Hm, I think we should not make this query even more complex (especially with orderings, they can be surprisingly slow). I think it's fine to only display things based on the filter implemented here - that is, simply not showing all reports labeled `invalid` with a probability > 0.95.

I'd prefer a separate view that allows us to have a look at lower probability issues and label them manually, but for SV's sake, we should start with >0.95 as a filter. This is already going to increase the number of visible reports, so I see no reason to include even more.. we're already kinda flooding them. 😅

Comment (ksy36) [reply src/server/helpers/user_reports_transform.ts]: Sounds good, I've removed the ORDER clause. Will look into adding a separate view in another PR :)

**Review by denschub — APPROVED** [review]

**Pull request merged**